lib/signs/utilities/audio.ex

```diff
@@ -65,27 +64,14 @@ defmodule Signs.Utilities.Audio do
               prediction.trip_id not in sign.announced_boardings &&
                 (announce_boarding?(sign, prediction) ||
                    (announce_arriving?(sign, prediction) &&
```

Comment:
Is it still appropriate to call this `announce_arriving?`

Reply:
Not entirely, but the config value is still named that, and I didn't want to try and refactor that as part of this change.